modules/mono/glue/Managed/Files/Color.cs

```diff
@@ -114,6 +264,11 @@ public Color Black
             }
         }
 
+        public static Color ColorN(string name)
+        {
+            return _namedColors[name];
```

Comment:
This implementation is also missing normalization of the name: https://github.com/godotengine/godot/blob/f12cb82e0f7b3348ab678322cf5e729b38861f1e/core/color.cpp#L421-L426

I would also prefer if we threw an IllegalArgumentException if the name is not found, rather than the KeyNotFoundException Dictionary throws.